litentry/pallets/identity-management/src/lib.rs

```diff
@@ -66,15 +66,15 @@ pub mod pallet {
 		type ManageOrigin: EnsureOrigin<Self::Origin>;
 		/// challenge code type
 		type ChallengeCode: Member + Parameter + Default + Copy + MaxEncodedLen;
-		/// maximum did length
+		/// maximum identity length
 		#[pallet::constant]
 		type MaxDidLength: Get<u32>;
```

Comment:
should we also rename this to **MaxIdentityLength** ?

Reply:
Yes, thanks.
Actually we don't need it anymore